Models/PointLightIntensityTest/README.body.md

```diff
@@ -0,0 +1,24 @@
+## Screenshot
+
+![screenshot](screenshot/screenshot-large.jpg)
+
+## Description
+
+This model tests various colors of point lights (via `KHR_lights_punctual`) that all share the same intensity (1.0).
+
```

Comment:
Please mention here that the exact asset appearance is subject to the renderer's IBL and exposure settings and thus may differ from the screenshot.

Reply:
Done.